### ISSUE TITLE
ncneofetch displays weird banding artifacts on Terminal Resize -  (Windows Command Prompt)

### ISSUE THREAD

**Issue by guilt** [dankamongmen/notcurses#2417]
Please include the following data:
* `export | egrep 'LANG|LC_CTYPE|TERM'`

```
Karthik@KVK-PC MINGW64 /c/Program Files (x86)/Notcurses
$ export | egrep 'LANG|LC_CTYPE|TERM'
declare -x TERM="xterm-256color"
```

* notcurses version (available from `notcurses-demo i`)

```
3.0.0
```
* terminal name + version

```
Windows Command Prompt
WinVer = 22000.348 Windows 11 21H2
Wrap Text Output on Resize is **ON**
```

* issue screenshot

![Screenshot 2021-12-02 040254](https://user-images.githubusercontent.com/195178/144419570-7f786db0-6817-4ffe-9e38-a841a9878e41.png)

I noticed that for `ncneofetch`, when `Wrap Text Output on Resize` is on (default setting), there seems to be an artificial banding
on some lines. Which persist when the terminal is brought back to original size. Not sure why the line isn't reset after that last character. 

Other programs/demo are working/wrapping okay.

**Comment by dankamongmen** [comment]
ahhh yes i noticed this myself. it's windows terminal that's doing the unwelcome stretching. this happens after `ncneofetch` has already exited, right? in that case, there's not much it can do about it at that time...but perhaps we could do something when we originally write to keep this from happening? maybe some explicit background set or something. i'll look into it.

**Comment by guilt** [comment]
Issue spotted in Windows Terminal as well:

```
Windows Terminal
WinVer = 22000.348 Windows 11 21H2
Version = 1.11.2921.0
```

![image](https://user-images.githubusercontent.com/195178/144420065-e44b4e6f-b3fd-412a-90b9-86f5cfac05ea.png)

**Comment by dankamongmen** [comment]
yep, all those programs are frontends on the common WinPTY ConHost. you'll almost certainly see that in any windows terminal in which notcurses can run.

**Comment by dankamongmen** [comment]
ahh btw it looks like you also have a 3970X? it's nice to meet another man of taste =].

**Comment by guilt** [comment]
haha, yes I do! :)

**Comment by dankamongmen** [comment]
> haha, yes I do! :)

https://nick-black.com/dankwiki/index.php/Schwarzgerat_II hell yeah, 3970X hot coder summer

**Comment by dankamongmen** [comment]
what i might do is file an issue over on the Microsoft Terminal project, asking what kind of workaround they suggest. it'd be wise to get their input on the entirety of their rewrapping/extending solution.

**Comment by guilt** [comment]
Nice, love your BOM for the build! Very detailed instructions indeed ... I have a 10Gig upgrade planned with multimode fiber soon. :) 

Going to the issue, I'm not a 100% sure if we reset the terminal in the end of those few lines or not, maybe this should be confirmed by us before we reach out to the WT team. Currently this is just a noobie bug report, will post details when I gather a smaller testcase.

**Comment by dankamongmen** [comment]
to verify that kind of thing, i usually redirect the output to a file and then run it through `xxd`. redirection of stdout to a file is currently not working on windows, though =\ (#2409). 

so let's see we've got that plane sitting by itself...yeah, we're not going to do any kind of color reset on the border. so that's definitely not yet explored. it may well resolve this, but i'd rather not have to issue a bunch of color controls if i can possibly avoid it.

**Comment by j4james** [comment]
FYI, this is a known issue in Windows Terminal. https://github.com/microsoft/terminal/issues/32
